phpcs.xml.dist

```diff
@@ -19,10 +19,21 @@
 		<exclude-pattern>tests/</exclude-pattern>
 	</rule>
 
+	<rule ref="WordPress">
+		<exclude name="WordPress.Files.FileName.NotHyphenatedLowercase" />
+		<exclude name="WordPress.Files.FileName.InvalidClassFileName" />
```

Comment:
Can we also exclude weird stuff like `WordPress.PHP.DisallowShortTernary`? :D 

Looks like WP added this rule because of some weird reasons about not being useful in their use-cases and trying not to scare beginners. https://make.wordpress.org/core/2019/07/12/php-coding-standards-changes/

Reply:
@AlexP11223 agreed, disallow short ternary does not make much sense now that we are on PHP 7.1.

Reply:
✔️ b8f35c8